docs/how-to-use-ui.md

```diff
@@ -0,0 +1,79 @@
+
+![Model Registry](./images/dashboard-modelregistry-create2.png)
+
+Provide all the database configuration and click "create", once the status shown as "Ready", you can click on kebab menu and select "manage access" where you can manage the user-group that is mentioned above. Please note that to add users to user group, you need to log into OpenShift console and use User Management screens. Otherwise you can add individual users to "add-user" section to grant access to individual access to the registry created. Please note currently all the access control is based on Kubernetes Authn/AuthZ, there is no options to include third party authentication or authorization systems. Once the access is defined, installation of Model Registry is complete.
+
+### Catalog of Models in Model Registry
+
+Client on left navigation item "Model Registry" on the Dashboard, to take you to screen that shows the catalog of models that are registered in Model Registry. If no models are registered it will show an empty list. If there are registered models you can click to see details of the models
+
+![Model Registry](./images/dashboard-modelregistry-catalog1.png)
+
+
+### Register a Model
+
+There are two different ways of registring model in the Model Registry. One using the button in this screen "Register Model" which should pop the following screen. Follow the entries on the screen to register the model. Please note that, Model Registry currently does not actually store the model but it will capture the location of the model where it is stored in a S3 store. First get hold of the model from hugging face or other places, save it into S3 bucket, and provide the details of S3 bucket and its credentials into Register dialog box under "Data Connection" section. Once registered the model will show under model catalog screen.
+
+![Model Registry](./images/dashboard-modelregistry-catalog1.png)
+
+it will bring this screen for registering a model
+
+![Model Registry](./images/dashboard-modelregistry-registermodel.png)
+
+Another way register the model is using [Python library](./getting-started.md). Note that when you register a model first time, it creates a model record and another record for a initial version that is defined. If you ever need to deploy a another iteration of the model after re-training try to add a new version of the model under the same original model record. So, a model record can contain any number of versions underneath. Each version is individually managed. The below screen shows when model is drilled down to version level.
```

Comment:
```suggestion
Another way register the model is using [Python library](./getting-started.md). Note that when you register a model the first time, it creates a Registered Model entry and related entries for the initial version that is defined. If you ever need to deploy another iteration of the model after re-training, try to add a new Model Version of the model under the original Registered Model entry. A Registered Model can contain any number of Model Versions underneath. Each version is individually managed. The below screen shows when model is drilled down to version level.
```

I would adopt the Logical Model jargon here, which is also reflected in the UI. wdyt?